wai-aria/state/basic.tentative.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
```

Comment:
ARIA editors avoid using `state` vs `property` unless understanding the context is critical in that specific instance of the usage. Preferring `attribute` instead when referring to content attributes. 

I would request we avoid `state` in the path name here for a similar reason... Some authors may think, "these similar attributes are ARIA properties, not states, so I need to break up the test file structure illogically to match this ARIA taxonomy oddity."

Reply:
> Are you suggesting "attribute"? 

At a minimum, I'm suggesting avoiding the use of `state` in the filed path `/wai-aria/state/basic.tentative.html‎` Since the spec makes an almost arbitrary or irrelevant distinction between "states" and "properties"... 

We could call the new directory props or properties if you prefer that over attrs/attributes.